ColorPairsUnittests.py

```diff
@@ -0,0 +1,13 @@
+from ColorPairsCreation import get_color_from_pair_number
+from ColorPairsCreation import get_pair_number_from_color
+
+def test_number_to_pair(pair_number,
+                        expected_major_color, expected_minor_color):
```

Comment:
Try including the input and expectation in the test function names. Do ideas for new tests emerge?